26-register_publication/443-creator_begins_registering_a_registration_in_the_wizard.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Creator begins registering a Registration in the Wizard
+  @443
+  Scenario Outline: Creator begins registering a Registration in the Wizard
+    Given Creator begins registering a Registration
+    And they have selected <Method> for starting the Wizard
+    When they click Start
+    Then they see the Wizard
+    Examples:
+      | Method                 |
+      | Link to registration    |

```

Comment:
```suggestion
      | Link to registration   |
```

Reply:
Anbefaler pluginen for Cucumber for formattering: alexkrechik.cucumberautocomplete